app/ui/src/Server.elm

```diff
@@ -0,0 +1,17 @@
+module Server exposing (send, listen)
+
+import WebSocket
+
+
+webSocketUrl =
+    "ws://localhost:3000/stream"
+
+
+send : String -> Cmd msg
+send input =
+    WebSocket.send webSocketUrl input
```

Comment:
We're passing some query string around. I think it would make more sense to have this module provide a functional interface to the API rather than having the rest of the app know how to make queries.

i.e. provide `getX`, `createY` functions that build (and maybe exec) the queries.